server/models/ClientRequest.js

```diff
@@ -183,26 +184,32 @@ class ClientRequest {
     });
   }
 
-  checkHash(options) {
-    const hashes = this.getEnsuredArray(options.hashes);
-    const stoppedHashes = hashes.filter(hash => {
-      return torrentService.getTorrent(hash).status.includes(torrentStatusMap.stopped);
-    });
-    const hashesToStart = [];
+  checkHash(userId, options) {
```

Comment:
Each instance of `ClientRequest` receives the `userId` when it was instantiated, and it is stored on the instance as property `userId`. So none of the instance methods need to receive `userId` as an argument, they can just access the property at `this.userId`.

Reply:
you're right